content/en/serverless/azure_app_services/azure_app_services_container.md

```diff
@@ -204,6 +204,15 @@ In your **App settings** in Azure, set the following environment variables:
 - `DD_SERVICE`: How you want to tag your service. For example, `sidecar-azure`
 - `DD_ENV`: How you want to tag your env. For example, `prod`
 - `DD_SERVERLESS_LOG_PATH`: Where you write your logs. For example, `/home/LogFiles/*.log` or `/home/LogFiles/myapp/*.log`
+- `DD_AAS_INSTANCE_LOGGING_ENABLED`: When enabled this automatically configures log collecting for the addition file path - `/home/LogFiles/*$COMPUTERNAME*.log`
+
+***NOTE:*** If your application has multiple instances, it is important to have the application's log filename to include the `COMPUTERNAME` environment variable. 
+This ensures that log tailing is not creating duplicated logs by multiple instances reading the same file.
+
+![image](https://github.com/user-attachments/assets/57d355e9-7cee-45bf-b1fa-940d6fcb3350)
+
+The above application settings need to be added to the sidecar through setting the environment variables of sidecar.
+Alternatively, the environment variables can be set in the main container using the ***Allow access to all app settings*** option.
```

Comment:
```suggestion
Alternatively, you can set the variable in the main container and enable the ***Allow access to all app settings*** option so that the sidecar can read it.
```